### PR TITLE
eva/eager: Try new strategy for SGE and Java incompatibility

### PR DESCRIPTION
**Pull request by jfy133**
This adds new profiles for default (assumed small), medium, and big data.

As well as memory increases for all processes, each profile adds additional multipliers for memory overhead.

### CODE REVIEW

File: conf/pipeline/eager/eva.config [hunk 1]
```diff
@@ -129,28 +129,166 @@ process {
...
+        withName: qualimap {
+          clusterOptions = { "-S /bin/bash -l h_vmem=${(task.memory.toGiga() * 3)}G,virtual_free=${(task.memory.toGiga() * 3)}G" }
+        }
+
+        withName: damageprofiler {
+          clusterOptions = { "-S /bin/bash -l h_vmem=${(task.memory.toGiga() * 3)}G,virtual_free=${(task.memory.toGiga() * 3)}G" }
+        }
+
+        withName: circularmapper {
+          clusterOptions = { "-S /bin/bash -l h_vmem=${(task.memory.toGiga() * 3)}G,virtual_free=${(task.memory.toGiga() * 2)}G" }
```
Comment (aidaanva) [inline conf/pipeline/eager/eva.config:256]: should this be 3?

File: conf/pipeline/eager/eva.config [hunk 2]
```diff
@@ -129,28 +129,166 @@ process {
...
+        }
+
+        withName: circulargenerator {
+          clusterOptions = { "-S /bin/bash -l h_vmem=${(task.memory.toGiga() * 3)}G,virtual_free=${(task.memory.toGiga() * 2)}G" }
```
Comment (aidaanva) [inline conf/pipeline/eager/eva.config:260]: shouldn't it be * 3?

**Review by aidaanva — CHANGES_REQUESTED** [review]
I left a few comments

**Review by TCLamnidis — APPROVED** [review]
LGTM

**Review by aidaanva — APPROVED** [review]

**Pull request merged**